perl.c

```diff
@@ -4,6 +4,7 @@
  *    Copyright (C) 1993, 1994, 1995, 1996, 1997, 1998, 1999, 2000, 2001
  *    2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009, 2010, 2011, 2012
  *    2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023
+ *    2024
```

Comment:
This is strictly not required for the test to pass, but it's reasonable to reflect the expected year in the file's comment too.